benchmark/gate_kernels.jl

```diff
@@ -0,0 +1,317 @@
+                suite["gates"][gate_str][(string(n_qubits), string(q), string(q2), "BraketSV")] =
+                    @benchmarkable py_sv.evolve([$py_gate]) setup =
+                        (py_sv = local_sv.StateVectorSimulation($n_qubits, 0, 1))
+            end
+            for (gate_str, gate, py_gate) in zip(
+                ["CNot", "CY", "CZ", "CV", "Swap", "ISwap", "ECR"],
+                [BraketSimulator.CNot(),
+                 BraketSimulator.CY(),
+                 BraketSimulator.CZ(),
+                 BraketSimulator.CV(),
+                 BraketSimulator.Swap(),
+                 BraketSimulator.ISwap(),
+                 BraketSimulator.ECR(),
+                ],
+                [
+                    gate_operations.CX([q, q2]),
+                    gate_operations.CY([q, q2]),
+                    gate_operations.CZ([q, q2]),
+                    gate_operations.CV([q, q2]),
+                    gate_operations.Swap([q, q2]),
+                    gate_operations.ISwap([q, q2]),
+                    gate_operations.ECR([q, q2]),
+                ],
+            )
+                suite["gates"][gate_str][(string(n_qubits), string(q), string(q2), "Julia")] =
+                    @benchmarkable BraketSimulator.apply_gate!($gate, sv, $q, $q2) setup =
+                        (sv = zeros(ComplexF64, $n_amps))
+                suite["gates"][gate_str][(string(n_qubits), string(q), string(q2), "BraketSV")] =
+                    @benchmarkable py_sv.evolve([$py_gate]) setup =
+                        (py_sv = local_sv.StateVectorSimulation($n_qubits, 0, 1))
+            end
+            for q3 in [mod(q+2, n_qubits)] #setdiff(0:n_qubits - 1, q, q2)
+                for (gate_str, gate, py_gate) in zip(
+                    ["CCNot", "CSwap"],
+                    [BraketSimulator.CCNot(), BraketSimulator.CSwap()],
+                    [gate_operations.CCNot([q, q2, 2]), gate_operations.CSwap([q, q2, 2])],
+                )
+                    suite["gates"][gate_str][(
+                        string(n_qubits),
+                        string(q),
+                        string(q2),
+                        string(q3),
+                        "Julia",
+                    )] = @benchmarkable BraketSimulator.apply_gate!($gate, sv, $q, $q2, $q3) setup =
+                        (sv = zeros(ComplexF64, $n_amps))
+                    suite["gates"][gate_str][(
+                        string(n_qubits),
+                        string(q),
+                        string(q2),
+                        string(q3),
+                        "BraketSV",
+                    )] = @benchmarkable py_sv.evolve([$py_gate]) setup =
+                        (py_sv = local_sv.StateVectorSimulation($n_qubits, 0, 1))
+                end
+            end
+        end
+    end
+end
+
+for n_qubits = 2:2:max_qubits(Val(:noise))
+    n_amps = 2^n_qubits
+    prob = 0.1
+    gamma = 0.2
+    for q in 0:0#n_qubits-1
```

Comment:
Is this temporarily commented out?

Reply:
Yes, again to avoid doing lots of redundant benchmarks.